subprojects/matrix-benchmarking-workloads/rhods-notebooks-ux/horreum_lts_store.py

```diff
@@ -24,8 +26,16 @@ def register_lts_metric(cluster_role, metric):
 def _parse_lts_dir(add_to_matrix, dirname, import_settings):
     with open(dirname / "data.json") as f:
         data = json.load(f)
```

Comment:
I think here, after parsing `data` from JSON, you should reshape it to something as close as possible to what the rest of this module expects, ie get rid of this distinction:
```
                if entry.is_lts:
                    res = entry.data['metrics'][metric]['data']
                else:
                    res = entry.results.metrics[name][metric]
```
to only have
```
 res = entry.results.metrics[name][metric]
```

Reply:
basically, in one function you're serializing/marshalling the original structure into a Horreum JSON payload, 
and here you want to do the reverse